src/app/components/applications/SSV/ValidatorSuccessScreen/ValidatorSuccessScreen.tsx

```diff
@@ -51,7 +51,7 @@ const ValidatorSuccessScreen = () => {
           <Grid item container className={classes.Wrapper}>
             <Grid item className={classes.Text}>Your new validator is managed by the following cluster:</Grid>
             <Grid container item className={classes.ClusterID}>
-              <Typography>Validator Cluster {longStringShorten(clusterHash, 4)}</Typography>
+              <Typography>Validator Cluster {longStringShorten(clusterHash, 4).slice(2)}</Typography>
```

Comment:
It is better to use more deterministic way of cutting first `0x` for the cases when there is no leading `0x`.
Correct way: `.replace(/^0x/, '');`

Reply:
done